spot_ros2_control/CMakeLists.txt

```diff
@@ -1,4 +1,11 @@
 cmake_minimum_required(VERSION 3.8)
+
+# This is here so we can use jthread from C++ 20
+set(CMAKE_CXX_STANDARD 20)
+
+# Taken from spot_driver CMakeLists
+list(APPEND CMAKE_PREFIX_PATH /opt/spot-cpp-sdk)
```

Comment:
@khughes-bdai meta: OK with this for now, but the Spot C++ SDK installation procedure should account for this, providing mechanisms to expose itself to local builds.